test/test-parse-expression-statement.cpp

```diff
@@ -1059,7 +1059,8 @@ TEST(test_parse, statement_beginning_with_async_or_let) {
     }
 
     {
-      spy_visitor v = parse_and_visit_statement(name + u8" => {body;};");
+      spy_visitor v =
+          parse_and_visit_statement(u8"(" + name + u8" => {body;})");
```

Comment:
Must fix: This test is trying to test the statement-starts-with-`async` and statement-starts-with-`let` code paths. Adding parentheses here reduces test coverage. Undo this change.